### ISSUE TITLE
[Feature Request] furigana on the page could be used to sort or filter dictionary entries

### ISSUE THREAD

**Issue by joriki** [FooSoft/yomichan#2029]
Currently, furigana on the page don't seem to influence which dictionary entries are shown and in which order. The entries that match the furigana should be at the top of the list, and there could be an option to show only those entries.

(Firefox 94.0.2, yomichan 21.10.31.1, [settings](https://github.com/FooSoft/yomichan/files/7670441/yomichan-settings-2021-12-07-17-31-10.txt))

Somewhat related: #2028

**Comment by toasted-nutbread** [comment]
You are correct that furigana on the source page is not currently scanned and thus not used in the lookup process. This is due to it being somewhat difficult to incorporate into the text extraction process and uncommon enough to not provide a significant improvement to the lookup/disambiguation process.

This is not to say there is no benefit to having it implemented, this is only to confirm and explain why it currently is this way. And you are correct that this would apply to #2028 also.
